README.md

```diff
@@ -22,45 +22,45 @@ This ontology matcher can be used to generate alignments between knowledgebases.
 
 ## OntoEmma program
 
-### OntoEmmaWrapper module
-The module `OntoEmmaWrapper` is used to run the OntoEmma program. The wrapper implements the OAEI-specified arguments:
+### Run OntoEmma (make an alignment between two KBs)
+To run OntoEmma, use `run_ontoemma.py`. The wrapper implements the OAEI-specified arguments:
 
+- -p \<model_type> (lr = logistic regression, nn = neural network)
+- -m \<pretrained_model>
 - -s \<source_ont> 
 - -t \<target_ont>
 - -i \<input_alignment>
 - -o \<output_alignment>
-- -m \<pretrained_model>
 
 Example usage: 
 
-`python OntoEmmaWrapper.py -s source_ont.owl -t target_ont.owl -i input_alignment.tsv -o output_alignment.tsv -m pretrained_model`
+`python run_ontoemma.py -p nn -m model_path -s source_ont.owl -t target_ont.owl -i input_alignment.tsv -o output_alignment.tsv`
 
 This script assumes that the model has been pre-trained, and uses *align* functions in `OntoEmma.py` accordingly.
 
-### OntoEmma module
-The module `OntoEmma` is used for accessing the training and alignment capabilities of the OntoEmma model.
+### Train OntoEmma (train a new model)
+To train an alignment model, use `train_ontoemma.py`. The wrapper takes the following arguments:
+
+- -p \<model_type> (lr = logistic regression, nn = neural network)
+- -m \<pretrained_model>
+- -c \<configuration_file>
+
+Example usage:
 
-The first argument specifies the mode of operation:
-- `mode` the mode of operation, see *train* and *align*
+`python train_ontoemma.py -p nn -m model_path -c configuration_file.json`
```

Comment:
Is the `-c` option used when the model is logistic regression, i.e., `-p lr`? I think the answer is no, but if it's yes then it's not clear what the config file should look like.

Reply:
I changed the lr to use a config file as well, just specifying where the training and development datasets are in json. I will describe further in the documentation.